private/rules/jetifier_maven_map.bzl

```diff
@@ -0,0 +1,126 @@
+jetifier_maven_map = {
```

Comment:
How did you generate this map? Can you add some docs/instructions/tool please?

Reply:
I took https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-master-dev/jetifier/jetifier/core/src/main/resources/default.config and manually modified it's `pomRules` section and inlined `versions["latestReleased"]`. I'll write that into the file.

Reply:
Added comments.